pyscriptjs/src/App.svelte

```diff
@@ -1,76 +1,5 @@
 <script lang="ts">
     import Tailwind from './Tailwind.svelte';
-    import { loadInterpreter } from './interpreter';
```

Comment:
I removed it because the whole logic was moved to `pyconfig.ts` and not used here anymore, if needed, other parts can import directly from `./interpreter` 

The reason I moved to pyconfig was that it helps isolate the whole logic around declaring and initializing runtimes somewhere more contained than just the App. Not sure if pyconfig is the best place for it to be, but it's better than before.

I think there's less and less need for us to have the svelte app... and this also helps us understand how much we depend on it. Atm, afaict, we are only relying on its stores.